lib/govuk_index/presenters/common_fields_presenter.rb

```diff
@@ -16,13 +16,18 @@ class CommonFieldsPresenter
     delegate_to_payload :publishing_app
     delegate_to_payload :rendering_app
     delegate_to_payload :search_user_need_document_supertype
-    delegate_to_payload :title
     delegate_to_payload :user_journey_document_supertype
 
     def initialize(payload)
       @payload = payload
     end
 
+    def title
+      payload_title = payload["title"]
+      payload_title.prepend("#{section_id} - ") if format == "hmrc_manual_section"
```

Comment:
Is `section_id` ever missing for this format?

Reply:
Dose this need to be format specific?

Can't we just append the `section_id` if it is present (which currently only happens for hmrc_manual_sections?

```[section_id, payload_title].compact.join(' - ')```

Reply:
Needs to be prepended. HMRC Manual Sections need them but Manual Sections don't seem to. We don't yet know for other formats which have sections.